cmd/assetsvc/postgresql_utils.go

```diff
@@ -71,12 +72,30 @@ func (m *postgresAssetManager) getPaginatedChartList(namespace, repo string, pag
 		repoQuery = strings.Join(clauses, " AND ")
 		repoQuery = "WHERE " + repoQuery
 	}
-	dbQuery := fmt.Sprintf("SELECT info FROM %s %s ORDER BY info ->> 'name' ASC", dbutils.ChartTable, repoQuery)
+
+	paginationClause := ""
+	if pageSize > 0 && pageNumber > 0 {
```

Comment:
`pageNumber` confused me a bit, since setting the number to 0 would disable the pagination. Added a comment above, it's better to fallback in that case to `1` so this condition is not necessary.

Reply:
The double check here is just for also support the same logic (page=0 disable pagination) also in the tests, where this method is directly invoked. 
I agree with your previous comment about explicitly handling the page=0 case.  I'm gonna update it!

Reply:
> The double check here is just for also support the same logic (page=0 disable pagination) also in the tests, where this method is directly invoked.

It's a bit weird that setting `pageNumber = 0` has different outputs in different places (either set it to 1 or disable pagination). IMO it would be more clear to either set it to 1 as well here or return an error if the input is not valid (with a comment explaining why this has been sanitized before).